apis/core.oam.dev/v1beta1/application_types.go

```diff
@@ -82,7 +82,7 @@ type ApplicationSpec struct {
 // Application is the Schema for the applications API
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
-// +kubebuilder:resource:categories={oam},shortName=app
+// +kubebuilder:resource:categories={oam},shortName={app,velaapp}
```

Comment:
why the short name didn't change the CRD file? Does it really work?

Reply:
```
go mod tidy
git --no-pager diff
diff --git a/charts/vela-core/crds/core.oam.dev_applications.yaml b/charts/vela-core/crds/core.oam.dev_applications.yaml
index 9d4fc00..82aa353 100644
--- a/charts/vela-core/crds/core.oam.dev_applications.yaml
+++ b/charts/vela-core/crds/core.oam.dev_applications.yaml
@@ -27,7 +27,6 @@ spec:
     plural: applications
     shortNames:
     - app
-    - velaapp
     singular: application
   scope: Namespaced
   versions:
diff --git a/charts/vela-minimal/crds/core.oam.dev_applications.yaml b/charts/vela-minimal/crds/core.oam.dev_applications.yaml
index 9d4fc00..82aa353 100644
--- a/charts/vela-minimal/crds/core.oam.dev_applications.yaml
+++ b/charts/vela-minimal/crds/core.oam.dev_applications.yaml
@@ -27,7 +27,6 @@ spec:
     plural: applications
     shortNames:
     - app
-    - velaapp
     singular: application
   scope: Namespaced
   versions:
git diff --quiet || (echo `date +%H:%M:%S` [FAIL] please run 'make reviewable' to include all changes && false)
19:35:09 [FAIL] please run make reviewable to include all changes
make: *** [Makefile:57: check-diff] Error 1
```
This is the result of the last CI which confuse me.  It seems that after I have changed the CRD,  the CI report the err.

Reply:
do you run `make reviewable`？

Reply:
the `make reviewable` will invoke go generate to generate CRD with the help of controller runtime. If it removes the like in the CRD, maybe that's not take affect to write like that.